cpp/ql/src/Likely Bugs/Likely Typos/FutileParams.qhelp

```diff
@@ -5,18 +5,25 @@
 
 
 <overview>
-<p>A function is called with arguments despite having an empty parameter list. This may indicate
-that the incorrect function is being called, or that the author misunderstood the function.</p>
+<p>A function is called with arguments despite having been <i>declared</i> with an empty parameter list.
+In addition, we were either unable to find a <i>definition</i> of the function, or found one with
+an incompatible number of parameters.</p>
+
+<p>This may indicate that an incorrect function is being called, or that the signature (parameter list)
+of the called function is not known to the author.</p>
 
 <p>In C, a function declared with an empty parameter list <code>()</code> is considered to have an unknown
 parameter list, and therefore can be called with any set of arguments. To declare a function
 which takes no arguments, you must use <code>(void)</code> as the parameter list in any forward declarations.
-In C++, either style of declaration indicates that the function accepts no arguments.</p>
+In C++, <code>()</code> means the same as <code>(void)</code>, i.e., indicates that the function accepts no
```

Comment:
Our style guide does not like "i.e.": https://wiki.semmle.com/pages/viewpage.action?pageId=17105939